Rubberduck.Interaction/IMessageBox.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Forms = System.Windows.Forms;
+
+namespace Rubberduck.Interaction
+{
+    public interface IMessageBox
+    {
+        void Notify(string text);
+        void NotifyError(string text, string caption);
+        void NotifyWarn(string text, string caption);
+        bool Question(string text, string caption);
+        bool Prompt(string text, string caption);
+        bool ConfirmYN(string text, string caption);
+        bool ConfirmYN(string text, string caption, bool suggestion);
```

Comment:
I think I'd rather that it was spelled out `ConfirmYesNo` - this has no change on the intellisense selection and make it more readable.